packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -455,6 +455,10 @@ export class SDKClient {
             if (sdkClientError.isGrpcTimeout()) {
                 throw predefined.REQUEST_TIMEOUT;
             }
+
+            if (sdkClientError.isInvalidTransactionBody()) {
```

Comment:
Add a `TRACE_LOG_INVALID_TRANSACTION_BODY` env variable that can be set to true but it defaulted to false.
If true then log